R/spelling.R

```diff
@@ -24,5 +24,5 @@ use_spell_check <- function(vignettes = TRUE,
   spelling::spell_check_setup(
     pkg = proj_get(), vignettes = vignettes, lang = lang, error = error
   )
-  ui_todo("Run {ui_code('devtools::check()')} to trigger spell check")
+  ui_bullets(c("_" = "Run {.run devtools::check()} to trigger spell check."))
```

Comment:
Not sure we want a full check running in console here? maybe `devtools::spell_check()`, or suggest check in build pane.

Reply:
I'm not holding myself to rethinking the _content_ of every message. Changing the mechanics is more than enough! So I'm going to leave this.